ui/webui/test/README.rst

```diff
@@ -243,3 +266,58 @@ development environment that's independent from the host::
 
     toolbox create --image quay.io/cockpit/tasks -c cockpit
     toolbox enter cockpit
+
+Web UI End to end tests
+=======================
+
+End to end tests are stored in ``./ui/webui/test/integration``
+together with tplib test cases and test plans (those are only needed when running
+the tests using Permian).
+
+End to end tests use one more level of abstraction, class `IntegrationTest <https://github.com/rhinstaller/anaconda/blob/anaconda-39.16-1/ui/webui/test/helpers/integration.py#L38>`_.
```

Comment:
as said with implementation internals, this does not belong here, rather in the mentioned class docstring. 

This file is supposed to guide the user on test execution mostly. For development it's better to look at existing tests.